axon-server-connector/src/main/java/org/axonframework/axonserver/connector/heartbeat/HeartbeatSource.java

```diff
@@ -0,0 +1,15 @@
+package org.axonframework.axonserver.connector.heartbeat;
+
+/**
+ * Represents the mechanism used to send an heartbeat message to AxonServer.
+ *
+ * @author Sara Pellegrini
+ * @since 4.2.1
+ */
+public interface HeartbeatSource {
```

Comment:
Why `Source`, isn't this a `Sender` (or something equivalent)?

Reply:
as agreed I replaced with `pulse()`